pkg/template/template.gen.go

```diff
@@ -31,6 +31,18 @@ import (
 	"istio.io/mixer/pkg/expr"
```

Comment:
My understanding was that we'd leave in the generated code for the well known templates to make it easier for adapter authors.